src/System.Private.CoreLib/shared/System/Globalization/PersianCalendar.cs

```diff
@@ -6,122 +6,77 @@
 
 namespace System.Globalization
 {
-    // Modern Persian calendar is a solar observation based calendar. Each new year begins on the day when the vernal equinox occurs before noon.
-    // The epoch is the date of the vernal equinox prior to the epoch of the Islamic calendar (March 19, 622 Julian or March 22, 622 Gregorian)
-
-    // There is no Persian year 0. Ordinary years have 365 days. Leap years have 366 days with the last month (Esfand) gaining the extra day.
-    /*
-     **  Calendar support range:
-     **      Calendar    Minimum     Maximum
-     **      ==========  ==========  ==========
-     **      Gregorian   0622/03/22   9999/12/31
-     **      Persian     0001/01/01   9378/10/13
-     */
-
+    /// <summary>
+    /// Modern Persian calendar is a solar observation based calendar. Each new year begins on the day when the vernal equinox occurs before noon.
+    /// The epoch is the date of the vernal equinox prior to the epoch of the Islamic calendar (March 19, 622 Julian or March 22, 622 Gregorian)
+    /// There is no Persian year 0. Ordinary years have 365 days. Leap years have 366 days with the last month (Esfand) gaining the extra day.
+    ///
+    ///  Calendar support range:
+    ///      Calendar    Minimum     Maximum
+    ///      ==========  ==========  ==========
+    ///      Gregorian   0622/03/22   9999/12/31
+    ///      Persian     0001/01/01   9378/10/13
+    /// </summary>
     public class PersianCalendar : Calendar
     {
         public static readonly int PersianEra = 1;
 
-        internal static long PersianEpoch = new DateTime(622, 3, 22).Ticks / GregorianCalendar.TicksPerDay;
+        private static long s_persianEpoch = new DateTime(622, 3, 22).Ticks / GregorianCalendar.TicksPerDay;
```

Comment:
Looks like a bunch of these statics could be marked readonly?

Reply:
Fixed